nrf_wifi/fw_if/umac_if/inc/default/fmac_structs.h

```diff
@@ -356,6 +357,27 @@ struct nrf_wifi_fmac_priv_def {
 #endif /* CONFIG_NRF700X_STA_MODE */
 };
 
+/**
```

Comment:
Should this be protected by #ifdef CONFIG_NRF700X_RAWDATA_TX ?

Reply:
done